class_materials/github_submit_pull_request.md

```diff
@@ -35,10 +35,12 @@ Once you have a change OR changes pushed to your Github repo, you will want to s
 
 **REMINDER**:  This leaves you in the original author's repo (Chalmer Lowe in this case) and not your own!
 
+![green sticky note](images/Sticky-Note-02-Green-300px.png)
+
```

Comment:
This is the main change. Please ignore the rest, as my editor auto remove white space trail.